src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/Items/xlf/EditorConfigFiles.BrowseObject.xaml.de.xlf

```diff
@@ -4,97 +4,97 @@
     <body>
       <trans-unit id="Rule|EditorConfigFiles|Description">
         <source>File Properties</source>
-        <target state="new">File Properties</target>
+        <target state="translated">Dateieigenschaften</target>
         <note />
       </trans-unit>
       <trans-unit id="Rule|EditorConfigFiles|DisplayName">
         <source>.editorconfig File</source>
-        <target state="new">.editorconfig File</target>
+        <target state="translated">EDITORCONFIG-Datei</target>
```

Comment:
I don't know German but I would think `.editorconfig` should be kept verbatim here. It is the culture-invariant name of a file. Other translations here keep it.

Reply:
This is the correct way how German and Russian translate file extension (.abc file -> ABC-Datei for German for example). So this is expected.